### DISCUSSION TITLE
how I can buy timetagger lifetime sub for other person?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [almarklein/timetagger#442] [Q&A]
how I can buy timetagger lifetime sub for other person? he has GH account and Gmail

**Comment by almarklein** [comment]
I can provide you with a redeem code, that you can give to any person, who can then redeem it (at https://timetagger.app/redeem) to get the subscription. You can pay via PayPal or Wire transfer. Please contact me at [support@timetagger.app](mailto:support@timetagger.app) to discuss the details.
